src/pipecat/audio/turn/smart_turn.py

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (c) 2024–2025, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+
+import io
+import os
+from typing import Dict
+
+import numpy as np
+import requests
+from loguru import logger
+
+from pipecat.audio.turn.base_smart_turn import BaseSmartTurn
+
+
+class SmartTurnAnalyzer(BaseSmartTurn):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.remote_smart_turn_url = os.getenv("REMOTE_SMART_TURN_URL")
```

Comment:
We should pass this as a `base_url` instead of an environment variable. Use the variable in the test file.

Reply:
or simply `url`

Reply:
done.